chrome-extension/src/content.js

```diff
@@ -49,9 +49,9 @@ const platforms = {
       document.querySelectorAll('input').forEach(i => i.addEventListener('input', () => {
         const a = parseInt(document.getElementById('atraffic').value, 10);
         const b = parseInt(document.getElementById('btraffic').value, 10);
-        const e = parseFloat(document.getElementById('expectedprop').value, 10);
```

Comment:
@heinrich333 because the new SRMcheck expects a number between 0 and 100 I had to update this. The statistician in me doesn't like it, but it works.